src/validations/rules/ssp.sch

```diff
@@ -2246,14 +2246,33 @@
                 role="error"
                 test="oscal:prop[@ns eq 'https://fedramp.gov/ns/oscal' and @name = 'users-external-future' and @value castable as xs:integer and @value cast as xs:integer ge 0]">A
                 FedRAMP SSP must specify the number of future external users.</sch:assert>
-            
+
             <sch:assert
                 diagnostics="has-this-system-component-diagnostic"
                 doc:guide-reference="Guide to OSCAL-based FedRAMP System Security Plans §5.4.6"
                 id="has-this-system-component"
                 role="error"
                 test="exists(oscal:component[@type eq 'this-system'])">A FedRAMP SSP must have a self-referential (i.e., to the SSP itself)
                 component.</sch:assert>
+            <sch:let
+                name="emailString"
+                value="'email', 'e-mail', 'electronic mail'" />
+            <sch:report
+                diagnostics="has-email-diagnostic"
+                id="has-email"
+                role="warning"
+                test="
+                    if (//*[lower-case(.) = $emailString])
```

Comment:
The document
```
<ssp>
    <p>email is used.</p>
</ssp>
```
assessed using the XPath expression
```
//*[lower-case(.) = ('email', 'e-mail', 'electronic mail')]
```
will be considered to not have the word "email".

The document
```
<ssp>
    <p>email</p>
</ssp>
```
would. But both documents mention email.